### PR TITLE
Login page and Landing Page

### PR DESCRIPTION
**Pull request by maxsebso**
So this was supposed to be 2 pr's but it is actually one cause I forgot L. Still need to add Icons and some styling but cannot due to ANTD Docs being down. Will get that done soon though. Here are pics:

Screen Shot 2021-02-14 at 11 27 17 PM
Screen Shot 2021-02-14 at 11 27 25 PM
Screen Shot 2021-02-14 at 11 27 39 PM

### CODE REVIEW

File: src/components/login-signup/LoginContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+/*
+const FormTitle = styled(Title)`
+  text-align: center;
+  margin: 24px 0px 0px 0px;
+`;
+
+const ChildrenContainer = styled.div`
+  max-width: 960px;
+`;*/
+
```
Comment (jackblanc) [inline src/components/login-signup/LoginContainer.tsx:17]: ❓

File: src/containers/home/Home.tsx [hunk 2]
```diff
@@ -1,29 +1,167 @@
...
+    opacity: 0.7;
+  }
+`;
+
 const Home: React.FC = () => {
+  const history = useHistory();
+
+  const fakeLink = (to: string) => {
+    history.push(to);
+  };
```
Comment (jackblanc) [inline src/containers/home/Home.tsx:40]: This function name is suspicious

File: src/containers/login/Login.tsx [hunk 3]
```diff
@@ -24,45 +43,83 @@ const Login: React.FC<LoginProps> = ({ tokens }) => {
...
-            <Paragraph>{tokens.error}</Paragraph>
-          )}
-          <Form.Item>
-            <Button type="primary" htmlType="submit">
-              Submit
-            </Button>
-          </Form.Item>
-        </Form>
+            </Paragraph>
+            */}
```
Comment (jackblanc) [inline src/containers/login/Login.tsx:97]: ❓

**Review by jackblanc — COMMENTED** [review]
Looks good, few questions about commented-out code and a function name.

File: src/containers/login/Login.tsx (hunk 3 above)
Comment (maxsebso) [reply src/containers/login/Login.tsx:97]: Oh shit I will fix

File: src/containers/home/Home.tsx (hunk 2 above)
Comment (maxsebso) [reply src/containers/home/Home.tsx:40]: I will change name lmaoo

File: src/components/login-signup/LoginContainer.tsx (hunk 1 above)
Comment (maxsebso) [reply src/components/login-signup/LoginContainer.tsx:17]: I will remove

**Review by jackblanc — COMMENTED** [review]
You requested another review but didn't change anything lol

**Comment by jackblanc** [comment]
also update your branch with master, merge conflicts

**Comment by maxsebso** [comment]
Screen Shot 2021-02-24 at 12 42 09 AM

**Comment by maxsebso** [comment]
added icons

File: src/components/index.tsx [hunk 4]
```diff
@@ -23,3 +23,15 @@ export const ContentContainer = styled.div`
   padding: 24px;
   margin: auto;
 `;
+
+export const Outer = styled.div`
+  background-color: #d4d9e7;
+  padding: 32px 32px 32px 32px;
```
Comment (rymaju) [inline src/components/index.tsx:29]: ⛏️ Can simplify to `padding: 32px`

File: src/components/index.tsx [hunk 5]
```diff
@@ -23,3 +23,15 @@ export const ContentContainer = styled.div`
...
+  border-radius: 5px;
+`;
+
+export const Inner = styled.div`
+  padding: 32px 32px 32px 32px;
```
Comment (rymaju) [inline src/components/index.tsx:34]: ⛏️ Can simplify to `padding: 32px`

File: src/components/navbar/NavBar.tsx [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+            <Col>
+              <Link
+                to="/"
+                onClick={() => {
+                  history.push('/');
+                }}
+              >
+                <Logo
+                  src="https://c4cneu-public.s3.us-east-2.amazonaws.com/Icons/C4C-Banner_Landing.jpeg"
+                  alt="LLB Logo"
```
Comment (rymaju) [inline src/components/navbar/NavBar.tsx:114]: ⛏️ Should probably be changed to HATS

File: src/components/navbar/index.tsx [hunk 7]
```diff
@@ -1,214 +1,23 @@
...
-  :hover {
-    color: ${PRIMARY};
-  }
-`;
-
-const ActiveNavBarButton = styled(NavBarButton)`
-  color: ${PRIMARY};
-  font-weight: 500;
+const NavRow = styled(Row)`
+  background-color: #294186;
```
Comment (rymaju) [inline src/components/navbar/index.tsx:6]: Should this be a constant?

File: src/containers/home/Home.tsx [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+            <Col span={12}>
+              <InContain
+                onClick={() => {
+                  link('/select-school');
+                }}
+              >
+                <Row>
+                  <Col span={8}>
+                    <FormOutlined
+                      style={{
```
Comment (rymaju) [inline src/containers/home/Home.tsx:75]: Should we move this into a styled component?

File: src/containers/login/index.tsx [hunk 9]
```diff
@@ -24,47 +42,69 @@ const Login: React.FC<LoginProps> = ({ tokens }) => {
...
+                      Submit
+                    </SubmitButton>
+                  </Center>
+                </Form.Item>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <Center>
+                  <Link to="/not-found" component={Typography.Link}>
```
Comment (rymaju) [inline src/containers/login/index.tsx:99]: Should these be route enums?

File: src/components/login-signup/LoginContainer.tsx (hunk 1 above)
Comment (jackblanc) [reply src/components/login-signup/LoginContainer.tsx:17]: Still not deleted

File: src/App.tsx [hunk 10]
```diff
@@ -96,11 +96,6 @@ const App: React.FC = () => {
                           exact
                           component={TrainingMentorshipInfo}
                         />
-                        <Route
```
Comment (jackblanc) [inline src/App.tsx:99]: why is this deleted?

File: src/components/navbar/index.tsx (hunk 7 above)
Comment (jackblanc) [reply src/components/navbar/index.tsx:6]: +1

File: src/components/navbar/index.tsx [hunk 11]
```diff
@@ -1,214 +1,23 @@
...
 `;
 
-const Logo = styled(Image)`
-  width: 300px;
-  margin: 16px;
+const HeadTitle = styled.h1`
+  font-weight: bold;
+  color: white;
```
Comment (jackblanc) [inline src/components/navbar/index.tsx:11]: use constant here too

File: src/containers/home/Home.tsx (hunk 8 above)
Comment (jackblanc) [reply src/containers/home/Home.tsx:75]: yes

File: src/containers/home/Home.tsx [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+    opacity: 0.7;
+  }
+`;
+
+const Home: React.FC = () => {
+  const history = useHistory();
+
+  const link = (to: string) => {
+    history.push(to);
+  };
```
Comment (jackblanc) [inline src/containers/home/Home.tsx:50]: There's no point in having this function

File: src/containers/home/Home.tsx [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+                }}
+              >
+                <Row>
+                  <Col span={8}>
+                    <UserOutlined
+                      style={{
+                        fontSize: '50px',
+                        marginTop: '14px',
+                        marginLeft: '5px',
+                      }}
```
Comment (jackblanc) [inline src/containers/home/Home.tsx:112]: styled component
